packages/calling-component-bindings/src/utils/callUtils.ts

```diff
@@ -75,3 +83,31 @@ const memoizedUpdateDisplayName = memoizeFnAll((participantId: string, participa
     return participant;
   }
 });
+
+/**
+ * @private
+ */
+export const isTeamsCallParticipants = (
+  participants: CommunicationIdentifier[]
+): participants is (PhoneNumberIdentifier | MicrosoftTeamsUserIdentifier | UnknownIdentifier)[] => {
+  for (const p of participants) {
+    if (isCommunicationUserIdentifier(p)) {
+      return false;
+    }
+  }
+  return true;
+};
+
+/**
+ * @private
+ */
+export const isNonTeamsCallParticipants = (
```

Comment:
The naming is a bit confusing, especially when compared with `isTeamsCallParticipants`.

Either rename this to `isAcsCallParticipants` or rename both: `containsNoAcsParticipant()` and `containsNoTeamsParticipant`. I like the latter because it is closer to what the function does and is grammatically correct ("is" should't be used with plural).

Reply:
I tried the latter name first, but found the logic is a bit hard to follow when reading it (because there are 2 big 'No's there), so I fallback to name of 'isAcsCallParticipants'